r/tests/testthat/test-compute-aggregate.R

```diff
@@ -37,7 +37,8 @@ test_that("sum.Array", {
 
   floats <- c(floats, NA)
   na <- Array$create(floats)
-  expect_identical(as.numeric(sum(na)), sum(floats))
+  # expect_identical(as.numeric(sum(na)), sum(floats))
+  expect_identical(as.numeric(sum(na)), NA_real_)
```

Comment:
Why these changes? If they have a specific purpose, there should be a comment explaining them.